crates/services/relayer/src/log.rs

```diff
@@ -97,6 +122,35 @@ impl TryFrom<&Log> for EthEventLog {
                     ),
                 })
             }
+            n if n == *config::ETH_FORCED_TX => {
+                if log.topics.len() != 1 {
```

Comment:
The `1` is a bit of a magic number. I looked up solidity topics and found:

> `topic[0]` always refers to the hash of the hash of the event itself, and can have up to 3 indexed arguments, which will each be reflected in the topics.

 Maybe it would be worth using a constant with a comment about why we use this value?

Reply:
added some comments for clarity